web/skins/classic/views/montage.php

```diff
@@ -204,7 +204,11 @@
     }
   }
 
-              echo getStreamHTML( $monitor, $monitor_options );
+              if ( $monitor->Type() == "WebSite" ) {
+                  echo getWebSiteUrl( 'liveStream'.$monitor->Id(), $monitor->Path(), reScale( $monitor->Width(), $scale ), reScale( $monitor->Height(), $scale ), $monitor->Name() );
```

Comment:
@connortechnology This is the line of code I am most concerned with. It almost certainly needs to be fixed. It is based on the previous way of doing things in montage. This is what I need to test.

Reply:
Well it looks like the old line needs to be the else condition...

Reply:
oh never mind, there it is below, looks fine to me.